translations/translation_fr.xml

```diff
@@ -317,15 +317,112 @@
+The position also roughly controls the starting point of your course.
+If you want to use the Giants helper to drive to the field, then you also need to set the target position close to the starting point of the course.
+The lane offset setting is only used if you want to have multiple helpers working on the same field. For this please checkout the separate help menu page below.
+		"/>
+		<text name="CP_help_page_readyJobMenuFunctions_text"											text="
+When the field position is placed correctly on a field, you will see the field border drawn on the map.
+If you are creating a CP: Fieldwork job, you will gain access to the course generator settings. 
+		"/>
+		<text name="CP_help_page_JobMenuExtras_text"													text="
+Now it's possible to start the driver directly from the menu. The Giants helper will drive to the target position and from there the CP job automatically takes over.
+Alternatively you can also start the driver from the HUD if you are close to the field with the vehicle or using the AutoDrive mod to deliver the CP job close to the field.
+		"/>
+		<!--CourseGenerator-->
+		<text name="CP_help_page_courseGenerator_title"													text="Course generator"/>
+		<text name="CP_help_page_courseGeneratorBase_text"												text="
+Every fieldwork course is generated by the course generator. This is a mighty tool, which allows you to customize the generated course to your needs.
+If you are new to this, you should probably start by only changing the headland amount setting.
+After that you can always change the other values and hit generate course and have a look, what impact certain values have on the course.
+Experimenting with it and check what impact the different setting on different fields have is the best way to get used to it.
+		"/>
+		<text name="CP_help_page_courseGeneratorBaseFunctions_text"										text="
+Basics:
+The first few values you see here, are real basic stuff.
+- Work width: Most tools get the exact work width detected and you won't have to care about this setting at all. Adjusting it, will have an impact on your overall course.
+- Multiple Tools: This setting is used when you want more then just one vehicle to work on your course. As this gets a bit more complicated, there is a separate help topic for it.
+- Number of Headlands: The best way to keep your vehicles on the field while turning at the end of each row is to add headlands.
+The Number of the Headlands multiplied by the work width should be at least the total length of your vehicle plus the attached tool.
+Center options:
+- Field center: There are different modes on how your field center pattern should look like. The classic and most used one is up/down.
+Spiral, Racetrack and Lands have their own specific advantages over the others. Lands for example will have the combine's pipe point out of the fruit most of the time to easier unload them.
```

Comment:
Could be nice to explain a bit more the purpose, pros and cons of each mode.

Reply:
Thanks for looking into it and give feedback.
I don't want to have it too long, but I take your feedback and try to figure something out.